cli/src/wallet_cli/mod.rs

```diff
@@ -196,6 +196,22 @@ pub enum WalletCommand {
     },
     /// Lists the implicit accounts of the wallet.
     ImplicitAccounts,
+    /// Adds a block issuer key to an account.
+    AddBlockIssuerKey {
+        /// The account to which the key should be added.
+        account_id: AccountId,
+        /// The hex-encoded public key to add.
+        // TODO: Use the actual type somehow?
+        block_issuer_key: String,
+    },
+    /// Removes a block issuer key from an account.
+    RemoveBlockIssuerKey {
+        /// The account from which the key should be removed.
+        account_id: AccountId,
+        /// The hex-encoded public key to remove.
+        // TODO: Use the actual type somehow?
+        block_issuer_key: String,
```

Comment:
You can use the `#[serde(with = "prefix_hex_bytes")]` tag with the specific type

Reply:
Like this you mean?
```rust
        #[serde(with = "prefix_hex_bytes")]
        block_issuer_key: BlockIssuerKey,
```

Fails with `error: cannot find attribute serde in this scope` because serde is not used on that struct.

Reply:
ah, you're right 😆 forget it!